### DISCUSSION TITLE
Provide a way to access fetched data by key

### DISCUSSION THREAD

**Discussion by lacorde** [nuxt/nuxt#16038] [Ideas]
Does Nuxt3 provides a way to access fetched data by key ?

As an example, Svelte's `useQueryClient` (whose purpose is similar to  `useAsyncData`), has a `queryCache` method to access/modify/replace cached data.

**Comment by wobsoriano** [comment]
Next release, probably. https://github.com/nuxt/framework/pull/9262

Basically, it's in

```ts
const nuxt = useNuxtApp()

nuxt.payload.data
```

**Reply by lacorde** [reply]
Will look out for this feature, thanks!
